Userland/Libraries/LibSoftGPU/Light/Light.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, Jesse Buhagiar <jooster669@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Vector4.h>
+
+namespace SoftGPU {
+
+struct Light {
+    bool is_on { false };
```

Comment:
```suggestion
    bool is_enabled { false };
```
Sounds better, no?

Reply:
It sure does, this was meant to invoke "turning a light on" but it sounds a bit awkward in reality.